contents/teams/feature-flags/objectives.mdx

```diff
@@ -1,54 +1,86 @@
-## Q3 2025 Objectives
+## Q4 2025 Objectives
 
-### Objective 1: Behavioral Cohort Targeting
+### Objective 1: Awesome documentation <TeamMember name="Dylan Martin" photo />
+Motivation: Present feature flags as a mature and sophisticated product through excellent documentation, following the experiments team's successful approach.
 
-Motivation: Enable dynamic behavioral targeting across PostHog – let customers target users based on what they've done (or haven't done) recently. In other words, enable customers to target feature flags on [behavioral cohorts](/docs/data/cohorts).
+What we will ship:
+* Partner with the [content team](/teams/content) for comprehensive refresh
+* Clear structure covering all SDK types and evaluation methods  
+* Real-world scenario walkthroughs
+* AI-related content – examples of how to use flags with AI features
+* Easy navigation from app to relevant docs
```

Comment:
basically cribbed this from [the experiments team](https://posthog.com/teams/experiments) – I'll talk to them next week about how it went and what they got out of it.